extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -506,33 +727,47 @@ export const HistoryItem = ({
   return (
     <div
       data-testid="history-item"
-      className="HistoryItem"
+      className="HistoryItem HistoryItem--gray12"
```

Comment:
I think it'd be nice to call this classname something more descriptive than `HistoryItem--gray12` as we may change the color to something other than `--gray12` at some point. For ex, something like `HistoryItem--highlight`